pages/change.php

```diff
@@ -19,6 +19,11 @@
 #
 #==============================================================================
 
+# missed defines in php 5
+define("LDAP_OPT_DIAGNOSTIC_MESSAGE", 0x0032);
+
```

Comment:
Would have prefer this:
````php
if ( !defined("LDAP_OPT_DIAGNOSTIC_MESSAGE") ) {
    define("LDAP_OPT_DIAGNOSTIC_MESSAGE", 0x0032);
}
````

And this code should be in index.php to avoid duplication in several files.

Reply:
May be it will better to define in lib/function.inc.php ?

Reply:
yes put it functions.inc.php please, so unit tests can use this constant too

Reply:
Done